src/main/resources/project/forms/deployment/DeleteVMForm.xml

```diff
@@ -0,0 +1,44 @@
+<!--
+
+     Copyright 2015 Electric Cloud, Inc.
+
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+
+-->
```

Comment:
This procedure definitely has value on its own that given a VM name and configuration, we can delete the VM from Azure cloud instance. But for dynamic environment, the requirement is to be able to take the resource name (or resource pool) and only the resource name and be able to delete both the VM as well as the resource using this information.
Since you would eventually need the configuration information in order to actually talk to Azure, one way to achieve this is to save the required information as properties on the resource at the time of provisioning. And then read them in the procedure that will delete the resource and the VM. So, my suggestion is to keep this procedure around but for dynamic envt, add another procedure 'teardown' which takes care of deleting the dynamic resource and VM. Refer to 'Teardown' and '_DeployDE' procedures in EC-OpenStack for how to implement this for Azure.